packages/main/src/TabContainer.hbs

```diff
@@ -66,18 +66,16 @@
 
 {{#*inline "contentArea"}}
 	<div class="{{classes.content}}" part="content">
-		{{#each items}}
-			{{#unless this.isSeparator}}
 				<div
 					class="ui5-tc__contentItem"
-					id="ui5-tc-contentItem-{{this._posinset}}"
-					?hidden="{{this.effectiveHidden}}"
+					id="ui5-tc-contentItem-{{this._selectedTab._posinset}}"
+					?hidden="{{this._selectedTab.effectiveHidden}}"
 					role="tabpanel"
```

Comment:
In the stripPresentation of each tab, add `aria-controls=selectedTabId`.
This is described in the Fiori Screen Reader specification:
Add an aria-controls ID relationship on the tabs to the
respective tabpanels. If there is only one tabpanel,
set all aria-controls relationships to the same
tabpanel ID.
Double check it in any case.

As an ID for the role=tabpanel element might not exist, check if you can add an ID to the slot element.